.github/workflows/check-links.yml

```
@@ -14,29 +14,33 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout the repository
-      uses: actions/checkout@v2.3.4
-      with:
-        path: repository
-
-    - name: Checkout the website
-      uses: actions/checkout@v2.3.4
-      with:
-        repository: seismo-learn/seismo-learn.github.io
-        ref: main
-        path: website
+    - name: Check out the repositories and sites
+      run: |
+        for repo in contributing links seismology seismology101 software website; do
```

Let's put links to the end until we fix all the invalid links.
```suggestion
        for repo in contributing seismology seismology101 software website links; do
```